src/components/HomePage.js

```diff
@@ -1,17 +1,74 @@
 import React from 'react';
 import '../styling/HomePage.css';
-import { Container, Row } from 'react-bootstrap';
+import { Button, Col, Container, Image, Row } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { LinkContainer } from 'react-router-bootstrap';
+import business1 from '../assets/business1.png';
+import business2 from '../assets/business2.png';
+import business3 from '../assets/business3.png';
+import business4 from '../assets/business4.png';
+import business5 from '../assets/business5.png';
+import volunteer1 from '../assets/volunteer1.png';
+import volunteer2 from '../assets/volunteer2.png';
+import volunteer3 from '../assets/volunteer3.png';
 
 const HomePage = () => (
   <Container fluid>
     <Row>
-      <div class="logotype">Build for Black Lives</div>
+      <Col className="logotype"> Build for Black Lives</Col>
     </Row>
     <Row>
-      <div class="bio-blurb">
-        We provide a platform to connect black-owned bbusinesses or organizations fighting racial injustice to
-        developers and designers who would like to support the cause with their technical skills.
-      </div>
+      <Col className="bio-blurb" xs={12} md={{ span: 6, offset: 3 }}>
+        We provide a platform to connect <span className="highlighted">black-owned businesses</span> or{' '}
+        <span className="highlighted">organizations fighting racial injustice</span> to{' '}
+        <span className="highlighted">developers</span> and <span className="highlighted">designers</span> who would
+        like to support the cause with their technical skills.
+      </Col>
+    </Row>
+    <Row className="button-row">
+      <Col className="text-center">
+        <LinkContainer to="/project-request">
+          <Button className="button" variant="warning" size="lg">
+            Request Project Help
+          </Button>
+        </LinkContainer>
+      </Col>
+      <Col className="text-center">
+        <LinkContainer to="/projects">
+          <Button className="button" variant="warning" size="lg">
+            Volunteer for Projects
+          </Button>
+        </LinkContainer>
+      </Col>
+    </Row>
+    <Row className="track-titles justify-content-center">Request Project Help</Row>
+    <Row className="justify-content-center">
+      <Image src={business1} thumbnail />
```

Comment:
you don't have to change this but a tip for future for less typing is if you just make an array with all the photo vars and do 
```
businessImages.map((img) => (<Image src={image}/>))
```

Reply:
good note! might refactor later haha (: